linuxbrew-build.sh

```diff
@@ -22,15 +22,20 @@ export HOMEBREW_NO_AUTO_UPDATE=1
 
 [[ -x ./bin/brew ]] || (echo "This script should be run inside Linuxbrew directory."; exit 1)
 
+echo
+echo "============================================================================================"
+echo "Building Linuxbrew in $PWD"
+echo "YB_USE_SSE4=$YB_USE_SSE4"
```

Comment:
We may consider using not SSE4, but CORE2_ARCH or some other naming because actually, this is not only about SSE4 but about old architecture support.

Reply:
I would prefer SSE4 to be the user-facing keyword, because it explicitly came up as the instruction set that we were not able to support when people tried to run YugaByte DB on older CPUs.